politeiawww/api/v1/api.md

```diff
@@ -164,7 +168,10 @@ Reply:
   "isadmin":false,
   "userid":"12",
   "email":"69af376cca42cd9c@example.com",
-  "publickey":"5203ab0bb739f3fc267ad20c945b81bcb68ff22414510c000305f4f0afb90d1b"
+  "publickey":"5203ab0bb739f3fc267ad20c945b81bcb68ff22414510c000305f4f0afb90d1b",
+  "paywalladdress":"",
```

Comment:
These should have actual values. If you're deriving this sample data from testing the live servers, you'll want to configure your `politeiawww.conf` with `paywallxpub` and `paywallamount`, like this:

```
paywallxpub=tpubVobLtToNtTq6TZNw4raWQok35PRPZou53vegZqNubtBTJMMFmuMpWybFCfweJ52N8uZJPZZdHE5SRnBBuuRPfC5jdNstfKjiAs8JtbYG9jx
paywallamount=10000000
```